include/eve/module/core/regular/fnms.hpp

```diff
@@ -29,16 +29,13 @@ namespace eve
     EVE_CALLABLE_OBJECT(fnms_t, fnms_);
   };
 
-//================================================================================================
-//! @addtogroup core_fma_family
+//======================================================================================================================
+//! @addtogroup core_fnms_family
```

Comment:
wrong group

Reply:
done

Reply:
the group is core_fma_family not core_fnms_family. This is still to be fixed